octopoes/octopoes/repositories/ooi_repository.py

```diff
@@ -292,9 +301,21 @@ def list_oois(
         limit: int = 20,
         scan_levels: set[ScanLevel] = DEFAULT_SCAN_LEVEL_FILTER,
         scan_profile_types: set[ScanProfileType] = DEFAULT_SCAN_PROFILE_TYPE_FILTER,
+        search_string: str | None = None,
+        order_by: Literal["scan_level", "object_type"] = "object_type",
+        asc_desc: Literal["asc", "desc"] = "asc",
     ) -> Paginated[OOI]:
         types = to_concrete(types)
 
+        search_statement = (
+            f"""[?e :xt/id ?id]
+                                [(clojure.string/includes? ?id \"{escape_string(search_string)}\")]"""
```

Comment:
would it make sense to (optionally) lowercase both the search_String in python, and in the database?

Reply:
```suggestion
                                [(clojure.string/includes? (clojure.string/lowercase ?id) \"{escape_string(search_string.lower())}\")]"""
```